src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -35,6 +39,7 @@ class Config {
     size_t m_loop_depth = 1;
 #ifdef SNIPPETS_DEBUG_CAPS
     PerfCountMode perf_count_mode = PerfCountMode::Disabled;
+    DebugCapsConfig debug_config;
```

Comment:
Since we introduced `DebugCapsConfig`, maybe it's worth to move `perf_count_mode` there?

Reply:
make sense, have moved `perf_count_mode `inside `DebugCapsConfig`. Thanks Vladislav!